src/main/java/org/abstractj/kalium/NaCl.java

```diff
@@ -87,6 +87,32 @@ public int crypto_generichash_blake2b_salt_personal(@Out byte[] buffer, @u_int64
                                                             @In byte[] salt,
                                                             @In byte[] personal);
 
```

Comment:
@lukin0110 good morning, I see that the following code is never used:

```
public static final int PWHASH_SCRYPTSALSA208SHA256_OPSLIMIT_SENSITIVE = 33554432;
public static final int PWHASH_SCRYPTSALSA208SHA256_SALTBYTES = 32;
public static final int PWHASH_SCRYPTSALSA208SHA256_MEMLIMIT_SENSITIVE = 1073741824;
```
Could you please remove and rebase your PR?